Neos.Fusion/Resources/Private/Fusion/Prototypes/Matcher.fusion

```diff
@@ -1,6 +1,9 @@
 prototype(Neos.Fusion:Matcher) {
   @class = 'Neos\\Fusion\\FusionObjects\\MatcherImplementation'
   condition = false
-  type = null
+  renderer = null
   renderPath = null
+  type = null
+  element {
```

Comment:
`element` should have a comment as it's use is always hard to explain

Reply:
sure - but only when we add all parameter docs to fusion - eg. in a later step as discussed here: https://github.com/neos/neos-development-collection/pull/3915#issuecomment-1274541752